doc/crypto/o2i_SCT_LIST.pod

```diff
@@ -18,20 +18,32 @@ decode and encode Signed Certificate Timestamp lists in TLS wire format
 
 =head1 DESCRIPTION
 
+The SCT_LIST and SCT functions are very similar to the i2d and d2i family of
+functions, except that they convert to and from TLS wire format, as described in
+RFC 6962. See L<d2i_SCT_LIST> for more information about how the parameters are
+treated and the return values.
 
+The SCT_signature functions are less similar. They still convert to and from
+TLS wire format, but they require that an SCT is provided that the signature is
+either taken from or inserted into.
 
 =head1 NOTES
 
-
+If an error occurs during a call to o2i_SCT_signature, the SCT may have its
+signature NID updated but not the signature value.
```

Comment:
Remove them from the public API.  Which means it is okay to just not document them for now.